web/plugin-manager/src/components/pluginBlock.tsx

```diff
@@ -0,0 +1,183 @@
+import { SetStateAction, useMemo, useState } from "react";
+import { ArrowPathIcon, TrashIcon, PlayCircleIcon } from "@heroicons/react/24/outline";
+import axiosServices from "../services/axios";
+import { AxiosResponse } from "axios";
+import { Plugin, PluginProps, PluginStatus } from "../../../shared/interfaces/IPlugin";
+import { statusHelper } from "../helpers/statusHelpers";
+
+function PluginBlock(p: PluginProps) {
+    // Callback to set error on parent
+    function sendErrorData(errorType: string, errorMessage: string) {
+        p.setErrorData(errorType, errorMessage);
+    }
+    // Data to display
+    let dataMemoized = useMemo(() => {
```

Comment:
Are you sure you want that the reader remember that "this data is memoized" or do you think it's better than the reader remember that these are plugin properties ?

Reply:
My question is not regarding the useMemo function (that I didn't know so thank you for this information :-)). it was about the variable name : dataMemoized. You should nema it to something usefull for the code reader. Something like `pluginProperties`, `pluginAttributes` or `pluginStatus` based on what is contained in this variable.

Reply:
Done